app/models/berkeley/terms.rb

```diff
@@ -61,9 +61,9 @@ def self.fetch(options = {})
       end
     end
 
-    def self.find_by_campus_solutions_id(term_id, options = {})
+    def self.find(campus_solutions_term_id, options = {})
```

Comment:
I'm pretty sure that moving forward we're going to be calling for terms by CS Term ID (aka `STRM`). This is a nicer interface.